sqlglot/parser.py

```diff
@@ -1131,9 +1131,15 @@ def _parse_select(self, nested=False, table=False):
                 limit=limit,
             )
             this.comments = comments
+
+            into = self._parse_into()
```

Comment:
how does this affect using into as an alias

Reply:
Good point, I'll look into it in a sec.

Reply:
```python
>>> sqlglot.parse_one("SELECT x AS INTO FROM bar").sql()
'SELECT x AS INTO FROM bar'
```